src/Relation/Binary/Properties/BoundedJoinSemilattice.agda

```diff
@@ -13,41 +13,9 @@ module Relation.Binary.Properties.BoundedJoinSemilattice
 
 open BoundedJoinSemilattice J
 
-open import Algebra.Definitions _≈_
-open import Data.Product
-open import Function.Base using (_∘_; flip)
-open import Relation.Binary
-open import Relation.Binary.Properties.Poset poset
+open import Relation.Binary.Lattice.Properties.BoundedJoinSemilattice
```

Comment:
This (and similar imports) needs to be opened `public`ly otherwise we'll break compatability.